src/main/resources/api/data-repository-openapi.yaml

```
@@ -2731,6 +2731,17 @@ components:
         description:
           type: string
           description: Free text that describes the profile
+        cloud_platform:
```

Can you make this camelCase for consistency?
```suggestion
        cloudPlatform:
```

---

Looking at other models in the yaml, I see both snake and camel case. Note that the openapi code generators create consistent names for the appropriate platforms; in python and in the swagger it's snake case, in java it's camel case.  So either convention we use here will end up consistent in the generated code. But it's probably good to be consistent with a single model at least.

---

Given the other properties of this class are camelCase, I'll change this one to match.

---

what the heck are you doing committing code!?  you're on vacation!